cumulusci/tasks/bulkdata/update_data.py

```diff
@@ -224,3 +226,19 @@ def _object_description(self, obj):
             return f'{obj} objects matching "{self.options["where"]}"'
         else:
             return f"all {obj} objects"
+
+
+class CumulusCIUpdatesApplication(SnowfakeryApplication):
+    """Takes over Snowfakery logging so CumulusCI can control it"""
+
+    MATCHER = re.compile(r"^Created [^ ]+.(csv|json)$")
+
+    def __init__(self, logger) -> None:
+        self.logger = logger
+        super().__init__()
+
+    def echo(self, message, *args, **kwargs):
+        # skip CSV creation messages
+        if not self.MATCHER.match(message):
+            print(repr(message))
+            self.logger.info(message)
```

Comment:
Shouldn't we leave whether to print up to how the logging system is configured?

Reply:
Sorry I was debugging the regexp and forgot to take out the print statement.

a69eb911